### ISSUE TITLE
Unused variable warning in C code

### ISSUE THREAD

**Issue by waternumbers** [waternumbers/tinyfiledialogsR#1]
The following is produced by the r-project mac builder
```
tinyfiledialogs.c:4046:7: warning: unused variable 'i' [-Wunused-variable]
                int i;
```

**Comment by vareille** [comment]
it is corrected in tinyfiledialogs version 3.17

**Comment by waternumbers** [comment]
Gone after updating to version 3.17.4 - thanks for flagging it
